src/python_routing_v02/troute/routing/compute.py

```diff
@@ -254,7 +254,7 @@ def compute_nhd_routing_v02(
 
         start_para_time = time.time()
         # if 1 == 1:
-        with Parallel(n_jobs=cpu_pool, backend="threading") as parallel:
+        with Parallel(n_jobs=cpu_pool, backend="loky") as parallel:
```

Comment:
changed Parallel backend to "loky" or process parallel. I observed that Process parallelization was substantially faster during CONUS simulations, when compared to thread parallel.